TTS/tts/utils/synthesis.py

```diff
@@ -5,19 +5,21 @@
 from torch import nn
 
 
-def numpy_to_torch(np_array, dtype, cuda=False):
+def numpy_to_torch(np_array, dtype, cuda=False, device="cpu"):
```

Comment:
Added new `device` argument to functions called in `Synthesizer`. To retain backwards compatibility, we keep the `cuda` argument for now; we should probably clean them up in the future and provide a single way of configuring device/enabling CUDA.